scripts/portstat

```diff
@@ -19,6 +21,9 @@ from natsort import natsorted
 from tabulate import tabulate
 from utilities_common.netstat import ns_diff, ns_brate, ns_prate, ns_util, table_as_json
 from utilities_common.intf_filter import parse_interface_in_filter
+from utilities_common.intf_filter import parse_interface_in_filter
```

Comment:
Duplicate line?

Reply:
This file was added to this PR by mistake. Removed the changes to this PR